src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5084,25 +5084,43 @@ Unify Enum declarations with other block type declarations.
     <!--********************
       -  TupleElementSyntax
       **********************-->
-    <node-structure name="TupleElementSyntax" parent="VisualBasicSyntaxNode">
+    <node-structure name="TupleElementSyntax" abstract="true" parent="VisualBasicSyntaxNode">
       <description>Represents a single declaration of a tuple element. </description>
       <grammar>TupleElement</grammar>
+    </node-structure>
 
-      <node-kind name="TupleElement"></node-kind>
+    <!--********************
+      -  TypedTupleElementSyntax
+      **********************-->
+    <node-structure name="TypedTupleElementSyntax" parent="TupleElementSyntax">
```

Comment:
UnnamedTupleElementSyntax sounds better

---
In reply to: [82260796](https://github.com/dotnet/roslyn/pull/14284#discussion_r82260796) [](ancestors = 82260796)